lib/MusicBrainz/Server/Data/Utils.pm

```diff
@@ -447,7 +447,7 @@ sub remove_lineformatting_characters {
     # - zwsp
     # - shy
     # - Other, control (including TAB \x09, LF \x0A, and CR \x0D)
-    =~ s/[\N{ZERO WIDTH SPACE}\N{SOFT HYPHEN}\p{Cc}]//gr;
+    =~ s/[\N{ZERO WIDTH SPACE}\N{SOFT HYPHEN}\N{HANGUL FILLER}\N{HALFWIDTH HANGUL FILLER}\N{HANGUL JUNGSEONG FILLER}\N{HANGUL CHOSEONG FILLER}\p{Cc}]//gr;
```

Comment:
Perhaps we should start using `/grxx` here to allow (ignored) whitespace/newlines within the character class.

Reply:
I also did consider having a separate line for the Hangul bit, maybe that'd be better.

Reply:
Is that a WIP or a PR ready to merge?

Reply:
It should be ready to merge, unless one of you two prefers splitting this into multiple lines or multiple regexes already now.

Reply:
Actually I just spotted an issue: The code comments above have to be updated as well.

Reply:
> Perhaps we should start using `/grxx` here to allow (ignored) whitespace/newlines within the character class.

@mwiencek: Not sure what you mean `/grxx`. A documentation link would be helpful.

Reply:
Specifically the `xx` flag: https://perldoc.perl.org/perlre#/x-and-/xx (the existing regexp is already using `gr`, hence `grxx`).

Reply:
I made some changes for clarity and future grow-ability, plus added `BRAILLE PATTERN BLANK` to the dodgy list.